Vagrantfile

```diff
@@ -55,7 +69,7 @@ Vagrant.configure("2") do |config|
   if $image_version != "current"
       config.vm.box_version = $image_version
   end
-  config.vm.box_url = "https://storage.googleapis.com/%s.release.core-os.net/amd64-usr/%s/coreos_production_vagrant.json" % [$update_channel, $image_version]
+  config.vm.box_url = "https://storage.googleapis.com/%s.release.core-os.net/amd64-usr/%s/coreos_production_vagrant_virtualbox.json" % [$update_channel, $image_version]
```

Comment:
Note: this URL only works for the alpha channel right now.

We should probably do something smarter.

Options include:

1. Wait for all channels to have the new vagrant_virtualbox image (don't support old versions)
1. 1., plus version parsing I guess?
1. Try both urls
1. ??